src/QualificationRunner.Core/Services/QualificationRunner.cs

```diff
@@ -118,19 +171,30 @@ private ObservedDataMapping copyObservedData(ObservedDataMapping observedDataMap
          };
       }
 
+      private static bool isLocalFileAndExists(string file)
+      {
+         try
+         {
+            return FileHelper.FileExists(file);

```

Comment:
This method crashes if the file is not formatted properly etc.. it can happen when we concatenate stuff together